### PR TITLE
added feature to run multiple database in one container

### PR DESCRIPTION
**Pull request by gaumire**

### CODE REVIEW

File: config [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
-export MYSQL_IMAGE=${MYSQL_IMAGE:="mysql"}
-export MYSQL_IMAGE_VERSION=${MYSQL_IMAGE_VERSION:="5.7.12"}
+export MYSQL_IMAGE=${MYSQL_IMAGE:="shrebo/mysql5.5"}
```
Comment (miraculixx) [inline config]: can we specify this in our dokku install as opposed to here? if yes we should do that, allowing us to contribute this PR back to upstream

**Comment by miraculixx** [comment]
looks good 
existing unit tests work ok

```
$ make test
...
77 tests, 0 failures
```
:+1: 

however, please add a unit test for extend/link/drop (see tests directory)

master is now rebased on upstream.

File: config (hunk 1 above)
Comment (gaumire) [inline config]: I think we can, we have to set the MYSQL_IMAGE env var then on our dokku instances I guess. I will have to check with it.

**Comment by gaumire** [comment]
@miraculixx Yes, existing tests pass. I will see to the tests for the added features. Also, we may now want to get import/export working too.

**Comment by miraculixx** [comment]
> get import/export 

good point, all in favor

**Pull request merged**